docs/maps/mapLine-series.md

```diff
@@ -25,3 +23,4 @@ This opens up many possibilities for the aforementioned drawing of non-standard
         }]
     });
 
+<iframe style="width: 100%; height: 520px; border: none;" src="https://highcharts.com/demo/maps/flight-routes" allow="fullscreen"></iframe>
```

Comment:
The SRC should point to the URL of the embed demo - like the top demo. Short info about what's this demo would be nice. Something like "A more advanced demo with customized line shapes to indicate flight routes.".